app/src/main/java/org/p2p/wallet/home/model/HomePresenterMapper.kt

```diff
@@ -118,4 +124,77 @@ class HomePresenterMapper(
             result.toList()
         }
     }
+
+    suspend fun mapToCellItems(
+        tokens: List<Token.Active>,
+        ethereumTokens: List<Token.Eth>,
+        strigaClaimableTokens: List<StrigaClaimableToken>,
+        visibilityState: VisibilityState,
+        isZerosHidden: Boolean
+    ): List<AnyCellItem> {
+        return withContext(dispatchers.io) {
```

Comment:
is this function really needs to be suspendable and run this logic inside IO dispatcher?